cmd/integration_test/build/buildspecs/cloudstack-test-eks-a-cli.yml

```diff
@@ -38,13 +38,15 @@ env:
     T_PRIVATE_REGISTRY_MIRROR_CA_CERT: "harbor-registry-data:authenticated_tinkerbell_caCert"
     T_AWS_IAM_ROLE_ARN: "aws-iam-auth-role:ec2_role_arn"
     T_CLOUDSTACK_DOMAIN: "cloudstack_ci_beta_connection:domain"
+    T_CLOUDSTACK_DOMAIN_2: "cloudstack_ci_beta_connection:domain_2"
     T_CLOUDSTACK_CREDENTIALS: "cloudstack_ci_beta_connection:credentials"
     T_CLOUDSTACK_CREDENTIALS_2: "cloudstack_ci_beta_connection:credentials_2"
     T_CLOUDSTACK_CREDENTIALS_3: "cloudstack_ci_beta_connection:credentials_3"
     T_CLOUDSTACK_ZONE: "cloudstack_ci_beta_connection:zone"
     T_CLOUDSTACK_ZONE_2: "cloudstack_ci_beta_connection:zone_2"
     T_CLOUDSTACK_ZONE_3: "cloudstack_ci_beta_connection:zone_3"
     T_CLOUDSTACK_ACCOUNT: "cloudstack_ci_beta_connection:account"
+    T_CLOUDSTACK_ACCOUNT_2: "cloudstack_ci_beta_connection:account_2"
```

Comment:
Can this be something like `T_CLOUDSTACK_ACCOUNT_FOR_MULTILEVEL_DOMAIN`. We can also update the secret names accordingly.

Reply:
Updated